src/mixins/allowable-actions-mixin.ts

```diff
@@ -127,7 +127,8 @@ export default class AllowableActionsMixin extends Vue {
       }
 
       case AllowableActions.PUT_BACK_ON: {
-        return this.isAllowedFiling(FilingTypes.PUT_BACK_ON)
+        return this.isAllowedFiling(FilingTypes.PUT_BACK_ON) &&
+        !!GetFeatureFlag('supported-put-back-on-entities')?.includes(this.getLegalType)
```

Comment:
As this line is a continuation of the previous line, please indent it.

While you're here, also please indent lines 121 and 126. Thanks!